core/services/ocr2/plugins/median/services.go

```diff
@@ -185,17 +187,17 @@ type medianContract struct {
 }
 
 type latestTransmissionDetailsResponse struct {
-	configDigest    ocr2types.ConfigDigest
-	epoch           uint32
-	round           uint8
-	latestAnswer    *big.Int
-	latestTimestamp time.Time
+	ConfigDigest    ocr2types.ConfigDigest
+	Epoch           uint32
+	Round           uint8
+	LatestAnswer    *big.Int
+	LatestTimestamp uint64
```

Comment:
What's the motivation behind this change? (time.Time to uint64)?

Reply:
because `abi.UnpackIntoInterface()` doesn't understand time.Time and the underlying onchain type maps to uint64. I will look through ryans encoding/decoding PRs, maybe there is a custom solution that would support time.TIme